pkgs/development/python-modules/markupsafe/default.nix

```
@@ -7,13 +7,13 @@
 
 buildPythonPackage rec {
   pname = "markupsafe";
-  version = "2.1.0";
+  version = "2.1.1";
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```

https://github.com/pallets/markupsafe/blob/2.1.1/setup.cfg#L36=